client/lib/products-values/constants.js

```diff
@@ -25,19 +26,27 @@ export const JETPACK_BACKUP_PRODUCTS = [
 
 // @TODO: Translate those strings once we have confirmed the copy.
 export const JETPACK_BACKUP_PRODUCT_SHORT_NAMES = {
-	[ PRODUCT_JETPACK_BACKUP_DAILY ]: 'Daily Backups',
-	[ PRODUCT_JETPACK_BACKUP_DAILY_MONTHLY ]: 'Daily Backups',
-	[ PRODUCT_JETPACK_BACKUP_REALTIME ]: 'Real-Time Backups',
-	[ PRODUCT_JETPACK_BACKUP_REALTIME_MONTHLY ]: 'Real-Time Backups',
+	[ PRODUCT_JETPACK_BACKUP_DAILY ]: translate( 'Daily Backups' ),
+	[ PRODUCT_JETPACK_BACKUP_DAILY_MONTHLY ]: translate( 'Daily Backups' ),
+	[ PRODUCT_JETPACK_BACKUP_REALTIME ]: translate( 'Real-Time Backups' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Real-time backups' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/jetpack-backup-translation)

Reply:
Really good suggestion, but this copy has already been finalized.